staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/cbor_test.go

```diff
@@ -399,6 +426,126 @@ func TestDecode(t *testing.T) {
 				}
 			},
 		},
+		{
```

Comment:
I noticed none of the decode tests actually contain the self-described CBOR tag in the test data. While that's not exactly in the scopes of this PR, should we also add some tests for that? It's technically just testing the imported decoder but might be worth it for those of us who are more paranoid.

Reply:
https://github.com/kubernetes/kubernetes/pull/124799